src/sif/dynamiccost.cc

```diff
@@ -232,6 +235,10 @@ Cost DynamicCost::BSSCost() const {
 }
 
 void DynamicCost::set_use_tracks(float use_tracks) {
+  // Calculate penalty value based on use preference. Return value
+  // in range [kMaxTrackPenalty; 0], if use < 0.5; or
+  // 0, if use > 0.5.
+  track_penalty_ = use_tracks < 0.5f ? (kMaxTrackPenalty * (1.f - 2.f * use_tracks)) : 0.f;
```

Comment:
Is this parameter documented somewhere? I couldn't find it in the [API docs](https://github.com/valhalla/valhalla/blob/master/docs/api/turn-by-turn/api-reference.md).

Its math for generating the track penalty is not very straight-forward. Eg: setting a value of `0.4` would mean applying a penalty of 1 min (?) to each track edge. CMIIW. Makes it a bit hard to reason as to what affect the value has without looking at the code.

Reply:
good catch, I forgot to add `use_tracks` option to docs( will do. the reasons why we use single option I described here https://github.com/valhalla/valhalla/pull/2845#discussion_r572716957